site/en/docs/extensions/mv3/getstarted/tut-focus-mode/index.md

```diff
@@ -0,0 +1,374 @@
+      await chrome.action.setBadgeText({
+        tabId: tab.id,
+        text: nextState,
+      });
+  ...
+```
+
+### Step 5: Add or remove the stylesheet {: #step-5 }
+
+Now it's time to change the layout of the page. Create a file named `focus-mode.css` and include the
+following code:
+
+{% Label %}focus-mode.css:{% endLabel %}
+
+```js
+body > .scaffold > :is(top-nav, navigation-rail, side-nav, footer),
+main > :not(:last-child),
+main > :last-child > navigation-tree,
+main .toc-container {
+  display: none;
+}
+
+main > :last-child {
+  margin-top: min(10vmax, 10rem);
+  margin-bottom: min(10vmax, 10rem);
+}
+```
+
+Great! Let's insert or remove the stylesheet using the [Scripting][api-scripting] API. Start by
+declaring the `"scripting"` permission in the manifest:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "permissions": ["activeTab", "scripting"],
+  ...
+}
+```
+
+{% Aside 'success' %}
+
+The Scripting API does not trigger a [permission warning][doc-perms-warning].
+
+{% endAside %}
+
+Finally, in `background.js` add the following code to change the layout of the page:
+
+{% Label %}background.js:{% endLabel %}
+
+```js
+  ...
+      if (nextState === "ON") {
+        // Insert the CSS file when the user turns the extension on
+        await chrome.scripting.insertCSS({
+          files: ["focus-mode.css"],
+          target: { tabId: tab.id },
+        });
+      } else if (nextState === "OFF") {
+        // Remove the CSS file when the user turns the extension off
+        await chrome.scripting.removeCSS({
+          files: ["focus-mode.css"],
+          target: { tabId: tab.id },
+        });
+      }
+    }
+  });
+```
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Can I use the Scripting API to inject code instead of a stylesheet?**
+{% endDetailsSummary %}
+
+Yes! You can use [`scripting.executeScript()`][api-scripting-es] to inject JavaScript. 
+
+{% endDetails %}
+
+### _Optional: Assign a keyboard shortcut_ {: #step-6 }
+
+Just for fun, let's add a shortcut to make it easier to enable or disable focus mode. Add the
+`“commands”` key to the manifest.
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "commands": {
+    "_execute_action": {
+      "suggested_key": {
+        "default": "Ctrl+U",
+        "mac": "Command+U"
+      }
+    }
+  }
+}
+```
+
+The `“_execute_action”` key runs the same code as the `action.onClicked()` event, so no additional
+code is needed!
+
+## Test that it works {: #try-out }
+
+Verify that the file structure of your project looks like the following: 
+
+```text
+Focus mode/
+├── manifest.json
+├── background.js
+├── focus-mode.css
+└── images/
+    ├── icon-16.png
+    ├── icon-32.png
+    ├── icon-48.png
+    └── icon-128.png
+```
+
+### Load your extension locally {: #locally }
+
+To load an unpacked extension in developer mode, follow the steps in [Development
+Basics][doc-dev-basics-unpacked].
+
+### Open an extension documentation page {: #open-sites }
```

Comment:
I think it might be good to tweak the name of this section to make it more obvious that we're testing the extension. Maybe something like "Test the extension on a documentation page"?

Reply:
Good call! changing now.